src/opnsense/mvc/app/models/OPNsense/Routes/Route/Migrations/M0_0_1.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace OPNsense\Routes\Route\Migrations;
+use OPNsense\Base\BaseModelMigration;
+class M0_0_1 extends BaseModelMigration
+{
+    public function run($model)
+    {
+       $model.save();
```

Comment:
@fabianfrz I don't think we can easily pull out the gateway list the way it is now, we did quite some cleanups some time ago, but there still is way too much logic in there. Ideally you should be able to read the current gateways either from the config. 

Another option could be to serialise (parts of) the legacy data using a configd call which hooks back into the model (so we can cleanly do it now and use legacy parts in the meantime),  we only might need to look at the caching mechanism in there.